solver_3.scad

```diff
@@ -18,7 +18,11 @@ use <belt.scad>;
 
 above_cube_holder() {
   color("red", 0.8) rubiks_cube_on_point();
-  color("green", 0.8) onto_point(cube_size) arm(include_gears=false);
+  color("green", 0.8) onto_point(cube_size) rotate(180, y) {
+    translate([cube_size/2+2/*TODO: this 2 represents arm.grip_width*/, 0, 0])
```

Comment:
for the todo, if it needs to be shared then either make arm.grip_width part of `constants.scad`, or in `arm.scad` add:

```
module offset_by_claw() {
  translate([cube_size/2 + grip_width, 0, 0]) children();
}
```

and here you'll be able to just do

```
offset_by_claw() arm();
claw();
```